src/launch/lib/j2props/j2props_utils.py

```diff
@@ -70,10 +69,10 @@ def __get_client(self):
         """
         if self._aws_client is None:
             # Create a Secrets Manager client
-            session = boto3.session.Session()
-            self._aws_client = session.client(
-                service_name="secretsmanager", region_name=self.region
+            session = boto3.session.Session(
+                region_name=self.region, profile_name="default"
```

Comment:
The way this is currently written, we are forced into using the default profile and a static region. If we don't specify an explicit `region_name` or `profile_name` here, I believe it will try to use the `AWS_REGION` and `AWS_PROFILE` env vars first if they exist and then fall back to the default, which is probably the behavior we actually want.

```suggestion
                region_name=self.region
```

This will give us the flexibility to be able to point the tool at alternate regions/profiles without code changes.